hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ResourceAttributeEnricher.java

```diff
@@ -23,6 +26,9 @@ public class ResourceAttributeEnricher extends AbstractTraceEnricher {
   private static final Logger LOGGER = LoggerFactory.getLogger(ResourceAttributeEnricher.class);
   private static final String RESOURCE_ATTRIBUTES_CONFIG_KEY = "attributes";
   private static final String NODE_SELECTOR_KEY = "node.selector";
+
```

Comment:
nit. remove extra spaces

run the gradle spotless apply   if not already done

Reply:
I did.. build wouldn't have passed without it